src/app/components/PriorityComplexityDisplay.tsx

```diff
@@ -0,0 +1,38 @@
+import { PriorityIcon } from "../icons/PriorityIcon";
+import { ComplexityIcon } from "../icons/ComplexityIcon";
+import { levelsCount } from "./LevelSelector";
+
+type PriorityComplexityDisplayProps = {
+  priority?: number;
+  complexity?: number;
+};
+
+export default function PriorityComplexityDisplay({
+  priority,
+  complexity,
+}: PriorityComplexityDisplayProps) {
+  const priorityComplexityDesc = (desc: number) => {
+    if (desc < 4) return "Low";
+    if (desc < 7) return "Medium";
+    return "High";
+  };
```

Comment:
This does not depend on state, you can move it out of the component or you can wrap it in useCallback so it does not get created again and again.

Reply:
Ok. I'll move the function outside component, and I'll merge it to main.